tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -376,6 +379,7 @@ def test_negative_add_image_vmware_with_invalid_name(self):
 
```

Comment:
Pls also check till the end of the module, i can't leave comments to lines not affected by this PR :) However we definitely need to update that status.